var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -13,6 +13,7 @@ class PyNumpy(Package):
     extends('python')
     depends_on('py-nose')
     depends_on('netlib-blas+fpic', when='+blas')
+    depends_on('netlib-blas+fpic+fortran', when='+blas+fortran')
```

Comment:
Not sure. The only reason I changed this part is to pass the `+fortran` variant down. Making a virtual package for blas is a separate branch.